### ISSUE TITLE
Highligth token endswith punctuation mark

### ISSUE THREAD

**Issue by aucan** [MaartenGr/KeyBERT#78]
https://github.com/MaartenGr/KeyBERT/blob/6ab9af1cfe74a126e709539a2467426d0881945c/keybert/_highlight.py#L54

Cannot find tokens ending in punctuation marks

**Comment by MaartenGr** [comment]
You are correct! They are a bunch more cases in fact where tokens will be missed. Especially if the n-gram is higher than 1. In practice, accounting for all those cases will require quite a number of changes to the code. Perhaps this is something for a future update but I cannot make any promises if it will be the next update or the one after that.
